samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START retail_purge_user_event]
+
+/*
+ * Purge user events into a catalog from inline source using Retail API
+ */
+
+package events;
+
+import static setup.SetupCleanup.writeUserEvent;
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.retail.v2.PurgeMetadata;
+import com.google.cloud.retail.v2.PurgeUserEventsRequest;
+import com.google.cloud.retail.v2.PurgeUserEventsResponse;
+import com.google.cloud.retail.v2.UserEventServiceClient;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class PurgeUserEvent {
+
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException {
+    String projectId = System.getenv("PROJECT_ID");
+    String defaultCatalog =
+        String.format("projects/%s/locations/global/catalogs/default_catalog", projectId);
+    String visitorId = "test_visitor_id";
+
+    writeUserEvent(visitorId);
```

Comment:
It looks like this is something that is used for testing? It doesn't need to be included in main, the test should set whatever resources it needs on the testing side.

Reply:
the visitor Id is a required value for the writeUserEvent method and is a filter value for the purgeUserEvent method using filtering
So it sud be here

Reply:
Changed constant visitorId value to random generated.

Reply:
It should have a placeholder value - it's failing because we're using the `main` for testing instead of calling `callPurgeUserEvents` directly for testing.

Reply:
Fixed.